dockertest/dockercmd_unittests.py

```diff
@@ -284,13 +282,13 @@ def test_no_fail_docker_cmd(self):
 
         docker_command = self.dockercmd.DockerCmd(self.fake_subtest,
                                                   'unittest_fail')
-        self.assertRaises(self.xceptions.DockerExecError,
+        self.assertRaises(self.output.xceptions.DockerExecError,
```

Comment:
Ya, that's just wrong (and I probably simply didn't care enough at the time).   Both the test and the code should be using the same exception, without exception :D  Otherwise, what good is checking?  Might as well compare it to an apple-pie.